src/fourier.jl

```diff
@@ -418,137 +428,106 @@ function avg_pds_from_iterable(
     end
 
     results = DataFrame()
-    results[!, "freq"] = freq
-    results[!, "power"] = cross
-    results[!, "unnorm_power"] = unnorm_cross
-    results = attach_metadata(
-        results,
-        (
-            n = n_bin,
-            m = n_ave,
-            dt = dt,
-            norm = norm,
-            df = 1 / (dt * n_bin),
-            nphots = n_ph,
-            mean = common_mean,
-            variance = common_variance,
-            segment_size = dt * n_bin,
-        ),
-    )
+    results.freq = freq
+    results.power = cross
+    results.unnorm_power = unnorm_cross
+
 
     return results
 end
 
-function avg_cs_from_iterables_quick(
-    flux_iterable1,
-    flux_iterable2,
-    dt::Real;
-    norm::String = "frac",
+function avg_cs_from_iterables_quick(flux_iterable1, flux_iterable2,
```

Comment:
This looks like it needs code formatting. Could you run JuliaFormatter over this?